src/backends/inspectdr.jl

```diff
@@ -22,6 +22,7 @@ const _inspectdr_attr = merge_with_base_supported([
     :foreground_color_legend, :foreground_color_title,
     :foreground_color_axis, :foreground_color_border, :foreground_color_guide, :foreground_color_text,
     :label,
+#    :seriescolor, :seriesalpha,
```

Comment:
`linecolor`, `fillcolor`, etc. fall back to `seriescolor`. The same is true for `...alpha`. So `seriescolor` and `seriesalpha` are supported.